src/Button/Button.test.tsx

```diff
@@ -30,7 +30,7 @@ describe('<Button />', () => {
 
     it('renders with props iconAfter and size', () => {
       const tree = renderer.create((
-        <Button iconAfter={Close} size="md">Button</Button>
+        <Button iconAfter={Close} size="sm">Button</Button>
```

Comment:
I just changed this to `sm` for some variation - too many of the tests were just using the same `md` size.